Time to play:
Loading lists of data
graph data Data(x=[81, 1], edge_index=[2, 324], y=[81], pos=[81, 2])
Traceback (most recent call last):
  File "/home/dami/Inverse_GNN/apply_GS.py", line 70, in <module>
    gsmat[i,:,:] = np.extend_dims(nmat[i,:,:].T @ transformedgraph[0,:,:].squeeze(),0)
  File "/home/dami/anaconda3/envs/fenics_d/lib/python3.10/site-packages/numpy/__init__.py", line 315, in __getattr__
    raise AttributeError("module {!r} has no attribute "
AttributeError: module 'numpy' has no attribute 'extend_dims'. Did you mean: 'expand_dims'?
Traceback (most recent call last):
  File "/home/dami/Inverse_GNN/train_FEMgpu.py", line 255, in <module>
    train_loss = train(fopt,epoch,lamda)
  File "/home/dami/Inverse_GNN/train_FEMgpu.py", line 172, in train
    with open("/home/dami/Inverse_GNN/FEM_output/gspdata","rb") as fh:
FileNotFoundError: [Errno 2] No such file or directory: '/home/dami/Inverse_GNN/FEM_output/gspdata'
(81, 81)
(1, 3, 31)
(100, 3, 81)
(100, 3, 81)
(1, 3, 31)